app/assets/stylesheets/homepage.css.scss

```diff
@@ -6,7 +6,15 @@
 #nav-bar {
     position:absolute;
     background-color: rgba(0,0,0,.4);
-    padding: 10px;
+    padding-top: 10px;
+
+  -moz-box-shadow:    inset 0px 5px 20px 0px rgba(0,0,0,0.3);
+  -webkit-box-shadow: inset 0px 5px 20px 0px rgba(0,0,0,0.3);
+  box-shadow:         inset 0px 5px 20px 0px rgba(0,0,0,0.3);
+}
+
+#nav-bar .mdl-layout__header-row {
```

Comment:
Avoid using id selectors<br>Merge rule `#nav-bar .mdl-layout__header-row` with rule on line 6<br>Selector `mdl-layout__header-row` should be written in lowercase with hyphens